src/swaps/bitcoin.rs

```diff
@@ -979,7 +1011,7 @@ impl BtcSwapTx {
             )?;
 
             // Step 7: Get boltz's partial sig
-            let refund_tx_hex = serialize(&refund_tx).to_lower_hex_string();
+            let refund_tx_hex = serialize(&refund_tx_cooperative).to_lower_hex_string();
             let partial_sig_resp = match self.swap_script.swap_type {
                 SwapType::Chain => {
                     boltz_api.get_chain_partial_sig(&swap_id, &pub_nonce, &refund_tx_hex)
```

Comment:
`get_chain_partial_sig` and `get_submarine_partial_sig` have `index` 0 hardcoded in them; we need to adjust those to have a parameter which needs to be the index of the input of the transaction

Reply:
Done: e3f9c573b132eceef96465bbd811a1cbea0cfc61